### ISSUE TITLE
SecurityError in FF 68 Mac

### ISSUE THREAD

**Issue by brandonkal** [tomhodgins/deqaf#1]
The issue is occuring on line 21 of index.js

![Image of Error](https://snag.gy/jQbny5.jpg)

Demo: https://codepen.io/tomhodgins/pen/aRWqro

**Comment by brandonkal** [comment]
After further investigation, I have identified the issue and why I was only seeing it in FF Mac.
This is a CORS issue that is thrown when the code cannot read a specific stylesheet. In my case, it was caused by the [Saka Key](https://github.com/lusakasa/saka-key) browser extension. Because deqaf is attempting to read the stylesheet injected by the extension, an error is thrown. This will also happen for stylesheets from other origins depending on the CORS configuration.

I would suggest wrapping in a try{} when attempting to read stylesheet.cssRules so it can reach the stylesheet that should be processed.

Awesome work by the way!
